# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-09-29 16:26-0300\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: api_app/apps.py:264
msgid "Lista de agentes activos de un Grupo"
msgstr "Active agents list for a Group"

#: api_app/apps.py:266
msgid "Campañas relacionadas a un supervisor"
msgstr "Campaigns related to a supervisor"

#: api_app/apps.py:268
msgid "Opciones de calificacion de una campaña (id campaña externo)"
msgstr "Campaign disposition options (external campaign id)"

#: api_app/apps.py:271
msgid "Opciones de calificacion de una campaña (id campaña oml)"
msgstr "Campaign disposition options (OML campaign id)"

#: api_app/apps.py:274
msgid "Ver/editar calificación cliente"
msgstr "View/edit Client Disposition"

#: api_app/apps.py:276
msgid "Crear calificacion cliente"
msgstr "Create Call Disposition"

#: api_app/apps.py:278
msgid "Crear contacto"
msgstr "Create Contact"

#: api_app/apps.py:280
msgid "Metadata de la base de datos de una campaña"
msgstr "Campaign database metadata"

#: api_app/apps.py:282
msgid "Crear nuevo rol"
msgstr "Create new Role"

#: api_app/apps.py:284
msgid "Eliminar rol"
msgstr "Delete Role"

#: api_app/apps.py:286
msgid "Actualizar permisos de Rol"
msgstr "Update Role Permissions"

#: api_app/apps.py:288
msgid "Devuelve información de los agentes en el sistema"
msgstr "Returns Agents information"

#: api_app/apps.py:291
msgid "Reporte de llamadas entrantes de supervisión."
msgstr "Supervision Inbound calls report"

#: api_app/apps.py:293
msgid "Reporte de llamadas salientes no dialer de supervisión."
msgstr "Supervision Outbound non dialer calls report"

#: api_app/apps.py:297
msgid "Ejecuta acciones de supervisión sobre agente (Deslogueo, pausas, etc..)"
msgstr "Apply supervision actions on Agent (Logoff, Pauses, etc..)"

#: api_app/apps.py:300
msgid "Cantidades por tipos de llamada en una campaña"
msgstr "Call types ammount for a Campaign"

#: api_app/apps.py:303
msgid "Cantidades por tipo de calificación en una campaña"
msgstr "Disposition types ammount for a Campaign"

#: api_app/apps.py:306
msgid "Reasignar una Agenda a otro Agente"
msgstr "Reassign Agenda to another Agent"

#: api_app/apps.py:308
msgid "Información de contacto de una  Agenda"
msgstr "Agenda's contact information"

#: api_app/apps.py:310
msgid "Exportar reporte de contactados de una campaña a csv"
msgstr "Export contacted campaign calls report to csv"

#: api_app/apps.py:313
msgid "Exportar reporte de calificados de una campaña a csv"
msgstr "Export dispositioned campaign calls report to csv"

#: api_app/apps.py:316
msgid "Exportar reporte de no atendidos de una campaña a csv"
msgstr "Export unnatended campaign calls report to csv"

#: api_app/apps.py:319 api_app/apps.py:322
msgid "Exportar reporte de calificaciones de una campaña a csv"
msgstr "Export dispositioned campaign calls report to csv"

#: api_app/apps.py:325
msgid "API para exportar resultados de base contactados a csv"
msgstr "API for exporting contacted database results to csv"

#: api_app/apps.py:328
msgid "Contactos de una campaña"
msgstr "Campaign contacts"

#: api_app/apps.py:330
msgid "Ejecuta un click 2 call"
msgstr "Start a click 2 call"

#: api_app/apps.py:332
msgid "Ejecuta el login del agente en Asterisk."
msgstr "Agent's Asterisk login"

#: api_app/apps.py:334
msgid "Ejecuta el logout del agente en Asterisk"
msgstr "Agent's Asterisk logout"

#: api_app/apps.py:336
msgid "Logout del agente en OML"
msgstr "Agent's OML logout"

#: api_app/apps.py:338
msgid "Pone al agente en una pausa"
msgstr "Start an Agent's pause"

#: api_app/apps.py:340
msgid "Saca al agente de una pausa"
msgstr "Stop an Agent's pause"

#: api_app/apps.py:342
msgid "Establece el estado Ringing del Agente"
msgstr "Set Agent's Ringing status"

#: api_app/apps.py:344
msgid "Establece el estado Reject del Agente"
msgstr "Save Agent's call reject log"

#: api_app/apps.py:346
msgid "Devuelve credenciales SIP de un agente"
msgstr "Return Agent's SIP credentials"

#: api_app/apps.py:348
msgid "Establece el estado de revisión de una Auditoría"
msgstr "Establishes review status for an Audit"

#: api_app/apps.py:351
msgid "Almacena en la base de datos los contactos subidos en archivo csv"
msgstr "Stores uploaded contact of a database in a csv file"

#: api_app/apps.py:354
msgid "Detecta si una llamada esta calificada."
msgstr "Detect if an call is qualified"

#: api_app/apps.py:357
msgid "Reenvía la llave de la instancia registrada por email"
msgstr "Sends registered instance key by email"

#: api_app/apps.py:359
msgid "Retorna el archivo de grabación especificado"
msgstr "Returns the specified recording file"

#: api_app/apps.py:362
msgid "Retorna Zip con archivos de grabación especificados"
msgstr "Returns .zip file with the specified recording files"

#: api_app/apps.py:365
msgid "Retorna URL del archivos de grabación asociado al callid"
msgstr "Returns the URL of the recording file associated to callid"

#: api_app/apps.py:368
msgid "Devuelve los contactos asignados de una campaña preview"
msgstr "Returns the assigned contacts of a preview campaign"

#: api_app/apps.py:371
msgid "Loggea el evento hold o unhold"
msgstr "Logs a hold or unhold event"

#: api_app/apps.py:374
msgid "Lista audios de asterisk"
msgstr "Asterisk audio list"

#: api_app/apps.py:377
msgid "Lista de grupos de agentes"
msgstr "Agent groups list"

#: api_app/apps.py:380
msgid "Lista de Agentes"
msgstr "Agent's list"

#: api_app/apps.py:383
msgid "Dashboard de supervision"
msgstr "Supervision dashboard"

#: api_app/apps.py:386
msgid "Auditoría a supervisores"
msgstr "Supervisor's Audit"

#: api_app/apps.py:389
msgid "Listas de nombres de columnas para Base de datos de Contactos"
msgstr "Contact Database fields names list"

#: api_app/apps.py:392
msgid "Reinicia servicio Wombat Dialer"
msgstr "Restart Wombat Dialer Service"

#: api_app/apps.py:395
msgid "Estado del servicio Wombat Dialer"
msgstr "Wombat Dialer service state"

#: api_app/apps.py:398
msgid "Lista de agentes por campaña"
msgstr "Campaign's agents list"

#: api_app/apps.py:401
msgid "Actualiza los agentes de una campaña"
msgstr "Update a Campaign's agents"

#: api_app/apps.py:404
msgid "Obtiene todos los agentes activos"
msgstr "Get all active agents"

#: api_app/apps.py:407
msgid "Obtiene las pausas que no estan eliminadas"
msgstr "Get non deleted pauses"

#: api_app/apps.py:410
msgid "Lista los conjuntos de pausas"
msgstr "List pause Sets"

#: api_app/apps.py:413
msgid "Obtiene el detalle de un conjunto de pausas"
msgstr "Get pause Set detail"

#: api_app/apps.py:416
msgid "Crea un conjunto de pausas"
msgstr "Create a pause Set"

#: api_app/apps.py:419
msgid "Actualiza un conjunto de pausas"
msgstr "Update pause Set"

#: api_app/apps.py:422
msgid "Elimina un conjunto de pausas"
msgstr "Delete pause Set"

#: api_app/apps.py:425
msgid "Crea una configuracion de pausa"
msgstr "Create pause configuration"

#: api_app/apps.py:428
msgid "Actualiza una configuracion de pausa"
msgstr "Update pause configuration"

#: api_app/apps.py:431
msgid "Elimina una configuracion de pausa"
msgstr "Delete pause configuration"

#: api_app/apps.py:434
msgid "Lista los sitios externos"
msgstr "External Site list"

#: api_app/apps.py:437
msgid "Obtiene el detalle de un sitio externo"
msgstr "Get External Site detail"

#: api_app/apps.py:440
msgid "Crea un sitio externo"
msgstr "Create external site"

#: api_app/apps.py:443
msgid "Actualiza un sitio externo"
msgstr "Update external site"

#: api_app/apps.py:446
msgid "Elimina un sitio externo"
msgstr "Deletes external site"

#: api_app/apps.py:449
msgid "Lista las autenticaciones para sitios externos"
msgstr "External Site authentications list"

#: api_app/apps.py:452
msgid "Obtiene el detalle de una autenticacion para sitio externo"
msgstr "Get External Site Authentication detail"

#: api_app/apps.py:455
msgid "Crea una autenticacion para sitio externo"
msgstr "Create external site authentication"

#: api_app/apps.py:458
msgid "Actualiza una autenticacion para sitio externo"
msgstr "Update external site authentication"

#: api_app/apps.py:461
msgid "Elimina una autenticacion para sitio externo"
msgstr "Deletes external site authentication"

#: api_app/apps.py:464
msgid "Oculta un sitio externo"
msgstr "Hides external site"

#: api_app/apps.py:467
msgid "Desoculta un sitio externo"
msgstr "Unhide external site"

#: api_app/apps.py:470
msgid "Lista las calificaciones"
msgstr "Call Disposition list"

#: api_app/apps.py:473
msgid "Crea una calificacion"
msgstr "Create Call Disposition"

#: api_app/apps.py:476
msgid "Actualiza una calificacion"
msgstr "Update call disposition"

#: api_app/apps.py:479
msgid "Elimina una calificacion"
msgstr "Delete call disposition"

#: api_app/apps.py:482
msgid "Obtiene detalle de una calificacion"
msgstr "Get call disposition detail"

#: api_app/apps.py:485
msgid "Lista los destinos entrantes"
msgstr "Destination list"

#: api_app/apps.py:488
msgid "Lista tipos de destinos entrantes"
msgstr "Destination type list"

#: api_app/apps.py:491
msgid "Lista los sistemas externos"
msgstr "External System list"

#: api_app/apps.py:494
msgid "Crea un sistema externo"
msgstr "Create external system"

#: api_app/apps.py:497
msgid "Actualiza un sistema externo"
msgstr "Update external system"

#: api_app/apps.py:500
msgid "Obtiene detalle de un sistema externo"
msgstr "Get External system detail"

#: api_app/apps.py:503
msgid "Obtiene los agentes para asignar a un sistema externo"
msgstr "Get Agents for External system assignment"

#: api_app/apps.py:506
msgid "Lista los formularios"
msgstr "Call Disposition Forms list"

#: api_app/apps.py:509
msgid "Crea un formulario"
msgstr "Create Call Disposition Form"

#: api_app/apps.py:512
msgid "Actualiza un formulario"
msgstr "Update call disposition form"

#: api_app/apps.py:515
msgid "Obtiene detalle de un formulario"
msgstr "Get call disposition form detail"

#: api_app/apps.py:518
msgid "Elimina un formulario"
msgstr "Delete call disposition"

#: api_app/apps.py:521
msgid "Oculta un formulario"
msgstr "Hides call disposition form"

#: api_app/apps.py:524
msgid "Desoculta un formulario"
msgstr "Unhide call disposition form"

#: api_app/apps.py:527
msgid "Lista las pausas"
msgstr "Pauses list"

#: api_app/apps.py:530
msgid "Crea una pausa"
msgstr "Create a pause"

#: api_app/apps.py:533
msgid "Actualiza una pausa"
msgstr "Update pause"

#: api_app/apps.py:536
msgid "Obtiene detalle de una pausa"
msgstr "Get pause detail"

#: api_app/apps.py:539
msgid "Elimina una pausa"
msgstr "Delete pause"

#: api_app/apps.py:542
msgid "Reactiva una pausa"
msgstr "Reactivate pause"

#: api_app/apps.py:545
msgid "Lista las rutas entrantes"
msgstr "Inbound route list"

#: api_app/apps.py:548
msgid "Crea una ruta entrante"
msgstr "Create Inbound route"

#: api_app/apps.py:551
msgid "Actualiza una ruta entrante"
msgstr "Update Inbound route"

#: api_app/apps.py:554
msgid "Obtiene detalle de una ruta entrante"
msgstr "Get Inbound route detail"

#: api_app/apps.py:557
msgid "Elimina una ruta entrante"
msgstr "Deletes Inbound route"

#: api_app/apps.py:560
msgid "Obtiene los destinos disponibles"
msgstr "Get available destinations list"

#: api_app/apps.py:563
msgid "Lista las rutas salientes"
msgstr "Outbound routes list"

#: api_app/apps.py:566
msgid "Crea una ruta saliente"
msgstr "Create Outbound route"

#: api_app/apps.py:569
msgid "Actualiza una ruta saliente"
msgstr "Update Outbound route"

#: api_app/apps.py:572
msgid "Obtiene detalle de una ruta saliente"
msgstr "Get Outbound route detail"

#: api_app/apps.py:575
msgid "Elimina una ruta saliente"
msgstr "Delete Outbound route"

#: api_app/apps.py:578
msgid "Obtiene las troncales sip para rutas salientes"
msgstr "Get SIP Trunks for Outbound routes"

#: api_app/apps.py:581
msgid "Obtiene las troncales huerfanas de una ruta saliente"
msgstr "Get orphan SIP Trunk for an Outbound route"

#: api_app/apps.py:584
msgid "Reordena las rutas salientes"
msgstr "Rearrange Outbound routes"

#: api_app/forms.py:53 api_app/forms.py:98
#: api_app/tests/tests_api_click2call.py:136
#: api_app/tests/tests_api_click2call.py:208
msgid "El agente no participa en la campaña."
msgstr "The agent is not related to campaign"

#: api_app/forms.py:62 api_app/tests/tests_api_click2call.py:144
msgid "El contacto no corresponde a la campaña."
msgstr "The contact is not related to campaign."

#: api_app/forms.py:100 api_app/tests/tests_api_click2call.py:174
#: api_app/tests/tests_api_click2call.py:196
msgid "El agente no corresponde al sistema externo."
msgstr "The agent is not related to external system."

#: api_app/services/base_datos_contacto_service.py:45
msgid "Ya existe una base de datos de contactos con ese nombre"
msgstr "A contact database with that name already exists"

#: api_app/services/base_datos_contacto_service.py:49
msgid ""
"El archivo especificado para realizar la importación de contactos no es "
"válido."
msgstr "The file specified to import contacts is not valid"

#: api_app/services/base_datos_contacto_service.py:54
#: api_app/tests/tests_api_subir_base_contactos.py:204
#: api_app/views/administrador.py:195
msgid "El archivo a procesar tiene nombres de columnas repetidos."
msgstr "The file to be processed has repeated column names."

#: api_app/services/base_datos_contacto_service.py:128
msgid "Archivo excede máximo de filas permitidas"
msgstr "File exceeds maximum rows allowed"

#: api_app/services/base_datos_contacto_service.py:132
msgid "Error al parsear el archivo csv"
msgstr "Error parsing csv file"

#: api_app/tests/campaign/agents/tests_add_agents_to_campaign.py:90
#: api_app/tests/campaign/agents/tests_add_agents_to_campaign.py:100
#: api_app/views/campaigns/add_agents_to_campaign.py:52
#: api_app/views/campaigns/add_agents_to_campaign.py:208
#: api_app/views/external_system.py:174
msgid "Se obtuvieron los agentes de forma exitosa"
msgstr "Agents fetched successfully"

#: api_app/tests/campaign/agents/tests_add_agents_to_campaign.py:119
#: api_app/views/campaigns/add_agents_to_campaign.py:143
msgid "Se agregaron los agentes de forma exitosa a la campaña"
msgstr "Agents to campaign added successfully"

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:70
#: api_app/views/pause_set.py:233
msgid "Se creo la configuracion de pausa de forma exitosa"
msgstr "Pause configuration created successfully"

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:88
#: api_app/views/pause_set.py:313
msgid "Se actualizo la configuracion de pausa de forma exitosa"
msgstr "Pause configuration updated successfully"

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:106
#: api_app/views/pause_set.py:280
msgid "Se elimino la configuracion de pausa de forma exitosa"
msgstr "Pause configuration deleted successfully"

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:125
#: api_app/views/pause_set.py:240
msgid "El timeout debe ser mayor a cero"
msgstr "Timeout must be greater than zero"

#: api_app/tests/pause_sets/tests_configuraciones_de_pausas.py:137
#: api_app/views/pause_set.py:288
msgid "No puedes dejar a un Conjunto de Pausas vacio"
msgstr "Pause set can't be empty"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:67
#: api_app/views/pause_set.py:70
msgid "Se obtuvieron los conjuntos de pausas de forma exitosa"
msgstr "Pause Sets fetched successfully"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:89
#: api_app/views/pause_set.py:129
msgid "Se creo el conjunto de pausa de forma exitosa"
msgstr "Pause set created successfully"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:110
#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:161
#: api_app/views/pause_set.py:134 api_app/views/pause_set.py:170
msgid "El nombre del conjunto es requerido"
msgstr "Set name required"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:126
#: api_app/views/pause_set.py:138
msgid "Debe existir al menos una pausa en el conjunto"
msgstr "Pause Set requires at least une Pause"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:144
#: api_app/views/pause_set.py:166
msgid "Se actualizo el conjunto de pausa de forma exitosa"
msgstr "Pause set updated successfully"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:173
#: api_app/views/pause_set.py:199
msgid "Se elimino el conjunto de pausa de forma exitosa"
msgstr "Pause set deleted successfully"

#: api_app/tests/pause_sets/tests_conjuntos_de_pausa.py:187
#: api_app/views/pause_set.py:206
msgid ""
"No puedes borrar un conjunto de pausas que esta asignado a un grupo de "
"agentes"
msgstr "Can't delete pause set already assigned to agent group"

#: api_app/tests/tests_api_call_dispositions.py:58
#: api_app/views/call_disposition.py:44
msgid "Se obtuvieron las calificaciones de forma exitosa"
msgstr "Call disposition fetched successfully"

#: api_app/tests/tests_api_call_dispositions.py:77
#: api_app/views/call_disposition.py:113
msgid "Se creo la calificacion de forma exitosa"
msgstr "Call disposition created successfully"

#: api_app/tests/tests_api_call_dispositions.py:93
#: api_app/views/call_disposition.py:120 api_app/views/call_disposition.py:163
msgid "Error al hacer la peticion"
msgstr "Error making request"

#: api_app/tests/tests_api_call_dispositions.py:113
#: api_app/views/call_disposition.py:155
msgid "Se actualizo la calificacion de forma exitosa"
msgstr "Call disposition updated successfully"

#: api_app/tests/tests_api_call_dispositions.py:129
#: api_app/views/call_disposition.py:193
msgid "Se elimino la calificacion de forma exitosa"
msgstr "Call disposition deleted successfully"

#: api_app/tests/tests_api_click2call.py:105
#: api_app/tests/tests_api_click2call.py:113
#: api_app/tests/tests_api_click2call.py:121
#: api_app/tests/tests_api_click2call.py:166
msgid "Escoja una opción válida. Esa opción no está entre las disponibles."
msgstr "Select a valid option. That option is not available."

#: api_app/tests/tests_api_click2call.py:158
#: api_app/tests/tests_api_contactos.py:170
#: api_app/tests/tests_api_contactos.py:262 api_app/views/agente.py:221
#: api_app/views/base.py:87 api_app/views/base.py:219
msgid "Sistema externo inexistente."
msgstr "Unkown external system"

#: api_app/tests/tests_api_contactos.py:114 api_app/views/base.py:176
msgid "Contacto agregado"
msgstr "Contact added"

#: api_app/tests/tests_api_contactos.py:142
#: api_app/tests/tests_api_contactos.py:160
#: api_app/tests/tests_api_contactos.py:229
#: api_app/tests/tests_api_contactos.py:252 api_app/views/base.py:118
#: api_app/views/base.py:251
msgid "No tiene permiso para editar la campaña."
msgstr "You don't have permission to edit the campaign"

#: api_app/tests/tests_api_contactos.py:178
#: api_app/tests/tests_api_contactos.py:195
#: api_app/tests/tests_api_contactos.py:272
#: api_app/tests/tests_api_contactos.py:292 api_app/views/base.py:111
#: api_app/views/base.py:244
msgid "Campaña inexistente."
msgstr "Campaign not found"

#: api_app/tests/tests_api_contactos.py:186
#: api_app/tests/tests_api_contactos.py:282 api_app/views/base.py:99
#: api_app/views/base.py:231
msgid "Debe indicar un idCampaign válido."
msgstr "You must provide a valid idCampaign"

#: api_app/tests/tests_api_contactos.py:302
msgid "Ya existe un contacto con ese id externo en la base de datos"
msgstr "Already exists a contact with this external id on the database"

#: api_app/tests/tests_api_contactos.py:334 api_app/views/base.py:135
msgid "El campo es obligatorio"
msgstr "The field is required"

#: api_app/tests/tests_api_contactos.py:345 api_app/views/base.py:127
msgid "Se recibieron campos incorrectos"
msgstr "Incorrect fields were received"

#: api_app/tests/tests_api_external_site_authentications.py:61
#: api_app/views/external_site_authentication.py:43
msgid "Se obtuvieron las autenticaciones para sitios externos de forma exitosa"
msgstr "External sites authentications fetched successfully"

#: api_app/tests/tests_api_external_site_authentications.py:74
#: api_app/views/external_site_authentication.py:142
msgid ""
"Se obtuvo la informacion de la autenticacion para sitio externo de forma "
"exitosa"
msgstr "External site authentication information fetched successfully "

#: api_app/tests/tests_api_external_site_authentications.py:105
#: api_app/views/external_site_authentication.py:168
msgid "Se elimino la autenticacion para sitio externo de forma exitosa"
msgstr "External site authentication deleted successfully"

#: api_app/tests/tests_api_external_site_authentications.py:121
#: api_app/views/external_site_authentication.py:174
msgid "No está permitido eliminar una autenticacion que tiene sitios externos"
msgstr "Authentications assigned to external sites can not be deleted"

#: api_app/tests/tests_api_external_site_authentications.py:148
#: api_app/views/external_site_authentication.py:70
msgid "Se creo la autenticacion para sitio externo de forma exitosa"
msgstr "External sites created successfully"

#: api_app/tests/tests_api_external_sites.py:67
#: api_app/views/external_site.py:44
msgid "Se obtuvieron los sitios externos de forma exitosa"
msgstr "External sites fetched successfully"

#: api_app/tests/tests_api_external_sites.py:80
#: api_app/views/external_site.py:141
msgid "Se obtuvo la informacion del sitio externo de forma exitosa"
msgstr "External site information fetched successfully "

#: api_app/tests/tests_api_external_sites.py:109
#: api_app/views/external_site.py:166
msgid "Se elimino el sitio externo de forma exitosa"
msgstr "External site deleted successfully"

#: api_app/tests/tests_api_external_sites.py:125
#: api_app/views/external_site.py:172
msgid "No está permitido eliminar un sitio externo asociado a una campaña"
msgstr "External sites assigned to campaigns can not be deleted"

#: api_app/tests/tests_api_external_sites.py:141
#: api_app/views/external_site.py:200
msgid "Se oculto el sitio externo de forma exitosa"
msgstr "External site hidden successfully"

#: api_app/tests/tests_api_external_sites.py:157
#: api_app/views/external_site.py:223
msgid "Se desoculto el sitio externo de forma exitosa"
msgstr "External sites successfully unhidden"

#: api_app/tests/tests_api_external_sites.py:182
#: api_app/views/external_site.py:71
msgid "Se creo el sitio externo de forma exitosa"
msgstr "External sites created successfully"

#: api_app/tests/tests_api_external_systems.py:59
#: api_app/views/external_system.py:45
msgid "Se obtuvieron los sistemas externos de forma exitosa"
msgstr "External systems fetched successfully"

#: api_app/tests/tests_api_external_systems.py:77
#: api_app/views/external_system.py:143
msgid "Se obtuvo la informacion del sistema externo de forma exitosa"
msgstr "External system information fetched successfully "

#: api_app/tests/tests_api_external_systems.py:106
#: api_app/views/external_system.py:73
msgid "Se creo el sistema externo de forma exitosa"
msgstr "External system created successfully"

#: api_app/tests/tests_api_external_systems.py:137
#: api_app/views/external_system.py:104
msgid "Se actualizo el sistema externo de forma exitosa"
msgstr "External system updated successfully"

#: api_app/tests/tests_api_forms.py:110 api_app/views/form.py:44
msgid "Se obtuvieron los formularios de forma exitosa"
msgstr "Call disposition form fetched successfully"

#: api_app/tests/tests_api_forms.py:131 api_app/views/form.py:147
msgid "Se obtuvo la informacion del formulario de forma exitosa"
msgstr "Call disposition form information fetched successfully "

#: api_app/tests/tests_api_forms.py:147 api_app/views/form.py:71
msgid "Se creo el formulario de forma exitosa"
msgstr "Call disposition form created successfully"

#: api_app/tests/tests_api_forms.py:163 api_app/views/form.py:178
msgid "Se elimino el formulario de forma exitosa"
msgstr "Call disposition form deleted successfully"

#: api_app/tests/tests_api_forms.py:177 api_app/views/form.py:184
msgid "No está permitido eliminar un formulario con calificaciones"
msgstr "Call dispositions forms with dispositions can not be deleted"

#: api_app/tests/tests_api_forms.py:193 api_app/views/form.py:212
msgid "Se oculto el formulario de forma exitosa"
msgstr "Call dispositions form hidden successfully"

#: api_app/tests/tests_api_forms.py:209 api_app/views/form.py:235
msgid "Se desoculto el formulario de forma exitosa"
msgstr "Call dispositions form successfully unhidden"

#: api_app/tests/tests_api_forms.py:235 api_app/views/form.py:102
msgid "Se actualizo el formulario de forma exitosa"
msgstr "Call disposition form updated successfully"

#: api_app/tests/tests_api_forms.py:257 api_app/views/form.py:108
msgid "No está permitido actualizar un formulario con calificaciones"
msgstr "Call dispositions forms with dispositions can not be updated"

#: api_app/tests/tests_api_inbound_route.py:78
#: api_app/tests/tests_api_outbound_route.py:115
#: api_app/views/inbound_route.py:47 api_app/views/outbound_route.py:50
msgid "Se obtuvieron las rutas entrantes de forma exitosa"
msgstr "Inbound Routes fetched successfully"

#: api_app/tests/tests_api_inbound_route.py:105
#: api_app/views/inbound_route.py:152
msgid "Se obtuvo la informacion de la ruta entrante de forma exitosa"
msgstr "Inbound Route information fetched successfully "

#: api_app/tests/tests_api_inbound_route.py:128
#: api_app/views/inbound_route.py:74
msgid "Se creo la ruta entrante de forma exitosa"
msgstr "Inbound Route created successfully"

#: api_app/tests/tests_api_inbound_route.py:147
#: api_app/views/inbound_route.py:183
msgid "Se elimino la ruta entrante de forma exitosa"
msgstr "Inbound Route deleted successfully"

#: api_app/tests/tests_api_inbound_route.py:193
#: api_app/views/inbound_route.py:108
msgid "Se actualizo la ruta entrante de forma exitosa"
msgstr "Inbound Route updated successfully"

#: api_app/tests/tests_api_outbound_route.py:139
#: api_app/views/outbound_route.py:153
msgid "Se obtuvo la informacion de la ruta saliente de forma exitosa"
msgstr "Outbound Route information fetched successfully "

#: api_app/tests/tests_api_outbound_route.py:161
#: api_app/views/outbound_route.py:77
msgid "Se creo la ruta saliente de forma exitosa"
msgstr "Outbound Route created successfully"

#: api_app/tests/tests_api_outbound_route.py:182
#: api_app/views/outbound_route.py:185
msgid "Se elimino la ruta saliente de forma exitosa"
msgstr "Outbound Route deleted successfully"

#: api_app/tests/tests_api_outbound_route.py:196
#: api_app/views/outbound_route.py:191
msgid "No está permitido eliminar una ruta saliente asociada a una campaña"
msgstr "Outbound Route assigned to campaigns can not be deleted"

#: api_app/tests/tests_api_outbound_route.py:358
#: api_app/views/outbound_route.py:111
msgid "Se actualizo la ruta saliente de forma exitosa"
msgstr "Outbound Route updated successfully"

#: api_app/tests/tests_api_pauses.py:66 api_app/views/pause.py:67
#: api_app/views/pause_set.py:45
msgid "Se obtuvieron las pausas de forma exitosa"
msgstr "Pauses fetched successfully"

#: api_app/tests/tests_api_pauses.py:84 api_app/views/pause.py:277
msgid "Se obtuvo la informacion de la pausa de forma exitosa"
msgstr "Pause information fetched successfully "

#: api_app/tests/tests_api_pauses.py:104 api_app/views/pause.py:93
msgid "Se creo la pausa de forma exitosa"
msgstr "Pause created successfully"

#: api_app/tests/tests_api_pauses.py:125 api_app/views/pause.py:198
msgid "Se elimino la pausa de forma exitosa"
msgstr "Pause deleted successfully"

#: api_app/tests/tests_api_pauses.py:139 api_app/views/pause.py:204
msgid "No está permitido eliminar un pausa con configuraciones creadas"
msgstr "Pauses with configurations can not be deleted"

#: api_app/tests/tests_api_pauses.py:165 api_app/views/pause.py:141
msgid "Se actualizo la pausa de forma exitosa"
msgstr "Pause updated successfully"

#: api_app/tests/tests_api_pauses.py:203 api_app/views/pause.py:241
msgid "Se reactivo la pausa de forma exitosa"
msgstr "Pause reactivated successfully"

#: api_app/tests/tests_api_permisos.py:56 api_app/views/administrador.py:83
msgid "Ya existe un rol con ese nombre"
msgstr "A Role with that name already exists"

#: api_app/tests/tests_api_permisos.py:66 api_app/views/administrador.py:78
msgid "Se esperaba el campo \"name\""
msgstr "Field \"name\" expected"

#: api_app/tests/tests_api_permisos.py:96 api_app/views/administrador.py:108
msgid "Se esperaba el campo \"role_id\" (numérico)"
msgstr "Field \"role_id\" (numeric) expected"

#: api_app/tests/tests_api_permisos.py:103
#: api_app/tests/tests_api_permisos.py:111
#: api_app/tests/tests_api_permisos.py:171 api_app/views/administrador.py:115
#: api_app/views/administrador.py:145
msgid "Id de Rol incorrecto"
msgstr "Wrong Role Id"

#: api_app/tests/tests_api_permisos.py:120
#: api_app/tests/tests_api_permisos.py:129 api_app/views/administrador.py:119
msgid "Se esperaba el campo \"permissions\" (lista)"
msgstr "Field \"permissions\" (list) expected"

#: api_app/tests/tests_api_permisos.py:139 api_app/views/administrador.py:124
msgid "Lista de permisos incorrecta"
msgstr "Wrong permission list"

#: api_app/tests/tests_api_permisos.py:162 api_app/views/administrador.py:150
msgid "No se puede borrar un rol asignado a usuarios."
msgstr "Deleting a Role assigned to a User is forbidden"

#: api_app/tests/tests_api_subir_base_contactos.py:101
#: api_app/views/administrador.py:192
msgid "falta parámetro filename en request"
msgstr "Filename parameter missing in request"

#: api_app/tests/tests_api_subir_base_contactos.py:113
#: api_app/tests/tests_api_subir_base_contactos.py:125
#: api_app/views/administrador.py:214
msgid "falta parámetro:"
msgstr "Missing parameter"

#: api_app/tests/tests_api_subir_base_contactos.py:138
#: api_app/views/administrador.py:229
msgid "campo de teléfono no coincide con nombre de columna"
msgstr "Phone field does not match column name"

#: api_app/tests/tests_api_subir_base_contactos.py:151
#: api_app/views/administrador.py:237
msgid "campo de id externo no coincide con nombre de columna"
msgstr "External id field does not match column name"

#: api_app/tests/tests_api_subir_base_contactos.py:163
#: api_app/views/administrador.py:189
msgid "la extensión del archivo no es .CSV"
msgstr "The file extension is not .CSV"

#: api_app/tests/tests_api_subir_base_contactos.py:177
#: api_app/views/administrador.py:185
#, python-brace-format
msgid "La longitud del nombre no debe exceder los {0} caracteres"
msgstr "The length of the name should not exceed {0} characters"

#: api_app/views/administrador.py:202
msgid "no se pudo crear la base de datos de contacto"
msgstr "Contact Database could not be created"

#: api_app/views/administrador.py:226
msgid "lista de campos teléfono vacia"
msgstr "List of fields phone empty"

#: api_app/views/administrador.py:281
#, python-brace-format
msgid "Error en el intento de conexion a: {0} debido {1}"
msgstr "Error in connection attempt to: {0} due to {1}"

#: api_app/views/administrador.py:286
msgid "Error en el servidor externo"
msgstr "Error on the external server"

#: api_app/views/agente.py:68
msgid "Error al generar sip password"
msgstr "Error creating sip password"

#: api_app/views/agente.py:220 api_app/views/agente.py:255
#: api_app/views/base.py:76 api_app/views/base.py:208
msgid "Hubo errores en los datos recibidos"
msgstr "There were errors on received data"

#: api_app/views/agente.py:246
msgid "Error al ejecutar la llamada"
msgstr "Error when executing the call"

#: api_app/views/agente.py:436
msgid "Auditoría inexistente"
msgstr "Audit not found"

#: api_app/views/agente.py:441
msgid "No tiene permiso para modificar la auditoría"
msgstr "You don't have permission to edit the audit"

#: api_app/views/call_disposition.py:55
msgid "Error al obtener las calificaciones"
msgstr "Error fetching call dispositions"

#: api_app/views/call_disposition.py:70
msgid "Se obtuvo el detalle de la calificacion de forma exitosa"
msgstr "Call disposition detail fetched successfully"

#: api_app/views/call_disposition.py:80 api_app/views/call_disposition.py:209
msgid "No existe la calificacion"
msgstr "Call disposition does not exist"

#: api_app/views/call_disposition.py:85
msgid "Error al obtener el detalle de la calificacion"
msgstr "Error fetching call disposition detail"

#: api_app/views/call_disposition.py:127
msgid "Error al crear la calificacion"
msgstr "Error creating call disposition"

#: api_app/views/call_disposition.py:171
msgid "No existe la calificacion que se quiere actualizar"
msgstr "call disposition does not exist"

#: api_app/views/call_disposition.py:177
msgid "Error al actualizar la calificacion"
msgstr "Error updating call disposition"

#: api_app/views/call_disposition.py:201
msgid "No está permitido eliminar una calificacion reservada del sistema"
msgstr "System reserved call dispositions can not be deleted"

#: api_app/views/call_disposition.py:214
msgid "Error al eliminar la calificacion"
msgstr "Error deleting call disposition"

#: api_app/views/campaigns/add_agents_to_campaign.py:64
#: api_app/views/supervisor.py:407 api_app/views/supervisor.py:422
msgid "No existe la campaña"
msgstr "Campaign does not exist"

#: api_app/views/campaigns/add_agents_to_campaign.py:88
#, python-brace-format
msgid "QueueAdd failed - agente: {0} de la campana: {1} "
msgstr "QueueAdd failed - agent: {0} from campaign: {1}"

#: api_app/views/campaigns/add_agents_to_campaign.py:117
#, python-brace-format
msgid "QueueRemove failed - agente: {0} de la campana: {1}"
msgstr "QueueRemove failed - agent: {0} from campaign: {1}"

#: api_app/views/campaigns/add_agents_to_campaign.py:128
msgid "No existe el agente en cola"
msgstr "Agent does not exist in queue"

#: api_app/views/campaigns/add_agents_to_campaign.py:150
msgid "No existe la campaña, no se pueden agregar agentes"
msgstr "Campaign does not exist, can't add agents"

#: api_app/views/campaigns/add_agents_to_campaign.py:166
msgid ""
"No existe el agente,                                        no se puede "
"eliminar de la member queue"
msgstr ""
"Agent does not exist,                                            can't "
"delete from member queue"

#: api_app/views/campaigns/add_agents_to_campaign.py:178
msgid ""
"No existe el agente,                                        no se puede "
"crear la member queue"
msgstr ""
"Agent does not exist,                                            can't "
"create member queue"

#: api_app/views/campaigns/add_agents_to_campaign.py:195
msgid "No se pudo confirmar la creación del dialplan"
msgstr "Could not confirm dialplan creation"

#: api_app/views/campaigns/add_agents_to_campaign.py:227
msgid "Error al obtener los agentes activos"
msgstr "Error fetching active agents"

#: api_app/views/external_site.py:54
msgid "Error al obtener los sitios externos"
msgstr "Error fetching external sites"

#: api_app/views/external_site.py:85
msgid "Error al crear el sitio externo"
msgstr "Error creating external site"

#: api_app/views/external_site.py:102
msgid "Se actualizo el sitio externo de forma exitosa"
msgstr "External site updated successfully"

#: api_app/views/external_site.py:119
msgid "No existe el sitio externo que se quiere actualizar"
msgstr "External site does not exist"

#: api_app/views/external_site.py:125
msgid "Error al actualizar el sitio externo"
msgstr "Error updating external site"

#: api_app/views/external_site.py:150
msgid "Error al obtener el detalle del sitio externo"
msgstr "Error fetching external site detail"

#: api_app/views/external_site.py:180
msgid "No existe el sitio externo"
msgstr "External site does not exist"

#: api_app/views/external_site.py:185
msgid "Error al eliminar el sitio externo"
msgstr "Error deleting external site"

#: api_app/views/external_site.py:208
msgid "Error al ocultar el sitio externo"
msgstr "Error hiding external site"

#: api_app/views/external_site.py:231
msgid "Error al desocultar el sitio externo"
msgstr "Error unhiding external site"

#: api_app/views/external_site_authentication.py:53
msgid "Error al obtener las autenticaciones para sitios externos"
msgstr "Error fetching external sites authentication"

#: api_app/views/external_site_authentication.py:85
msgid "Error al crear la autenticacion para sitio externo"
msgstr "Error creating external site authentication"

#: api_app/views/external_site_authentication.py:102
msgid "Se actualizo la autenticacion para sitio externo de forma exitosa"
msgstr "External site authentication updated successfully"

#: api_app/views/external_site_authentication.py:120
msgid "No existe la autenticacion para sitio externo que se quiere actualizar"
msgstr "External site authentication does not exist"

#: api_app/views/external_site_authentication.py:126
msgid "Error al actualizar la autenticacion para sitio externo"
msgstr "Error updating external site authentication"

#: api_app/views/external_site_authentication.py:152
msgid "Error al obtener el detalle de la autenticacion para sitio externo"
msgstr "Error fetching external site authentication detail"

#: api_app/views/external_site_authentication.py:182
msgid "No existe la autenticacion para sitio externo"
msgstr "External site authentication does not exist"

#: api_app/views/external_site_authentication.py:187
msgid "Error al eliminar la autenticacion para sitio externo"
msgstr "Error deleting external site authentication"

#: api_app/views/external_system.py:56
msgid "Error al obtener los sistemas externos"
msgstr "Error fetching external systems"

#: api_app/views/external_system.py:87
msgid "Error al crear el sistema externo"
msgstr "Error creating external system"

#: api_app/views/external_system.py:121
msgid "No existe el sistema externo que se quiere actualizar"
msgstr "External system does not exist"

#: api_app/views/external_system.py:127
msgid "Error al actualizar el sistema externo"
msgstr "Error updating external system"

#: api_app/views/external_system.py:152
msgid "No existe el sistema externo para obtener el detalle"
msgstr "External system does not exist"

#: api_app/views/external_system.py:158
msgid "Error al obtener el detalle del sistema externo"
msgstr "Error fetching external system detail"

#: api_app/views/external_system.py:184
msgid "Error al obtener los agentes"
msgstr "Error fetching agents"

#: api_app/views/form.py:54
msgid "Error al obtener los formularios"
msgstr "Error fetching call disposition forms"

#: api_app/views/form.py:85
msgid "Error al crear el formulario"
msgstr "Error creating call disposition form"

#: api_app/views/form.py:125
msgid "No existe el formulario que se quiere actualizar"
msgstr "Call disposition form does not exist"

#: api_app/views/form.py:131
msgid "Error al actualizar el formulario"
msgstr "Error updating call disposition form"

#: api_app/views/form.py:156
msgid "No existe el formulario para obtener el detalle"
msgstr "Call disposition form does not exist"

#: api_app/views/form.py:162
msgid "Error al obtener el detalle del formulario"
msgstr "Error fetching call disposition form detail"

#: api_app/views/form.py:192
msgid "No existe el formulario"
msgstr "Call disposition form does not exist"

#: api_app/views/form.py:197
msgid "Error al eliminar el formulario"
msgstr "Error deleting call disposition form"

#: api_app/views/form.py:220
msgid "Error al ocultar el formulario"
msgstr "Error hiding call disposition form"

#: api_app/views/form.py:243
msgid "Error al desocultar el formulario"
msgstr "Error unhiding call disposition form"

#: api_app/views/grabaciones.py:95
msgid "Exportación de grabaciones Zip en proceso"
msgstr "Exportation of recordings to a .Zip file in process"

#: api_app/views/inbound_route.py:57
msgid "Error al obtener las rutas entrantes"
msgstr "Error fetching Inbound routes"

#: api_app/views/inbound_route.py:80
msgid ""
"Se creo la ruta entrante pero no se pudo cargar la configuración telefónica"
msgstr "Inbound route created, but configuration was not loaded properly"

#: api_app/views/inbound_route.py:91
msgid "Error al crear la ruta entrante"
msgstr "Error creating inbound route"

#: api_app/views/inbound_route.py:119 api_app/views/inbound_route.py:196
msgid ""
"Se actualizo la ruta entrante pero no se pudo cargar la configuración "
"telefónica"
msgstr "Inbound route updated, but configuration was not loaded properly"

#: api_app/views/inbound_route.py:130
msgid "No existe la ruta entrante que se quiere actualizar"
msgstr "Inbound Route does not exist"

#: api_app/views/inbound_route.py:136
msgid "Error al actualizar la ruta entrante"
msgstr "Error updating inbound route"

#: api_app/views/inbound_route.py:161
msgid "No existe la ruta entrante para obtener el detalle"
msgstr "Inbound route does not exist"

#: api_app/views/inbound_route.py:167
msgid "Error al obtener el detalle de la ruta entrante"
msgstr "Error fetching inbound route detail"

#: api_app/views/inbound_route.py:190
msgid ""
"No está permitido eliminar una Ruta Entrante asociada con una campaña que "
"tiene una Ruta Saliente."
msgstr "Inbound routes assigned to campaigns can not be deleted"

#: api_app/views/inbound_route.py:202
msgid "No existe la ruta entrante"
msgstr "Inbound route does not exist"

#: api_app/views/inbound_route.py:207
msgid "Error al eliminar la ruta entrante"
msgstr "Error deleting Inbound route"

#: api_app/views/inbound_route.py:222
msgid "Se obtuvo la informacion de los destinos de forma exitosa"
msgstr "Destinations information fetched successfully "

#: api_app/views/inbound_route.py:241
msgid "Error al obtener los destinos de las rutas entrantes"
msgstr "Error fetching Inbound routes destinations"

#: api_app/views/outbound_route.py:60
msgid "Error al obtener las rutas salientes"
msgstr "Error fetching Outbound routes"

#: api_app/views/outbound_route.py:83
msgid ""
"Se creo la ruta saliente pero no se pudo cargar la configuración telefónica"
msgstr "Outbound route created, but configuration was not loaded properly"

#: api_app/views/outbound_route.py:94
msgid "Error al crear la ruta saliente"
msgstr "Error creating Outbound route"

#: api_app/views/outbound_route.py:120
msgid ""
"Se actualizo la ruta saliente pero no se pudo cargar la configuración "
"telefónica"
msgstr "Outbound route updated, but configuration was not loaded properly"

#: api_app/views/outbound_route.py:131
msgid "No existe la ruta saliente que se quiere actualizar"
msgstr "Outbound route does not exist"

#: api_app/views/outbound_route.py:137
msgid "Error al actualizar la ruta saliente"
msgstr "Error updating Outbound route"

#: api_app/views/outbound_route.py:163
msgid "No existe la ruta saliente para obtener el detalle"
msgstr "Outbound route does not exist"

#: api_app/views/outbound_route.py:169
msgid "Error al obtener el detalle de la ruta saliente"
msgstr "Error fetching Outbound route detail"

#: api_app/views/outbound_route.py:196
msgid "Se elimino la ruta pero no se pudo cargar la configuración telefónica"
msgstr "Outbound route deleted, but configuration was not loaded properly"

#: api_app/views/outbound_route.py:202 api_app/views/outbound_route.py:262
msgid "No existe la ruta saliente"
msgstr "Outbound route does not exist"

#: api_app/views/outbound_route.py:207
msgid "Error al eliminar la ruta saliente"
msgstr "Error deleting Outbound route"

#: api_app/views/outbound_route.py:222
msgid "Se obtuvieron las troncales sip de forma exitosa"
msgstr "SIP Trunks fetched successfully"

#: api_app/views/outbound_route.py:232
msgid "Error al obtener las troncales"
msgstr "Error fetching SIP Trunks"

#: api_app/views/outbound_route.py:247
msgid ""
"Se obtuvieron las troncales huerfanas de la ruta saliente de forma exitosa"
msgstr "Outbound route's Orphan trunks fetched successfully"

#: api_app/views/outbound_route.py:267
msgid "Error al obtener las troncales huerfanas de la ruta saliente"
msgstr "Error fetching Outbound route's Orphan trunks"

#: api_app/views/outbound_route.py:282
msgid "El orden de las rutas salientes se actualizo de forma exitosa"
msgstr "Outbound routes order updated successfully"

#: api_app/views/outbound_route.py:288
msgid "El orden es un campo requerido"
msgstr "Order is a required field"

#: api_app/views/outbound_route.py:293 api_app/views/outbound_route.py:298
msgid "El orden debe ser una lista de numeros enteros"
msgstr "Order must be an integer list"

#: api_app/views/outbound_route.py:303
msgid "El nuevo orden no coincide con la cantidad de rutas saliente"
msgstr "Order does not match the number of outboud routes"

#: api_app/views/outbound_route.py:310
msgid "No puede haber orden repetido en las rutas salientes"
msgstr "Outbound route's order can not repeat'"

#: api_app/views/outbound_route.py:332
msgid "Error al actualizar el orden de las rutas salientes"
msgstr "Error updating Outbound routes order"

#: api_app/views/pause.py:77 api_app/views/pause_set.py:55
msgid "Error al obtener las pausas"
msgstr "Error fetching pauses"

#: api_app/views/pause.py:107 api_app/views/pause.py:158
#: api_app/views/pause.py:215 api_app/views/pause.py:251
msgid "Error al sincronizar la pausa con asterisk"
msgstr "Error sinchronizing pauses with Asterisk"

#: api_app/views/pause.py:124
msgid "Error al crear la pausa"
msgstr "Error creating pauses"

#: api_app/views/pause.py:182
msgid "Error al actualizar la pausa"
msgstr "Error updating pause"

#: api_app/views/pause.py:226
msgid "Error al eliminar la pausa"
msgstr "Error deleting pause"

#: api_app/views/pause.py:262
msgid "Error al reactivar la pausa"
msgstr "Error reactivating pause"

#: api_app/views/pause.py:292
msgid "Error al obtener el detalle de la pausa"
msgstr "Error fetching pause detail"

#: api_app/views/pause_set.py:81
msgid "Error al obtener los conjuntos de pausas"
msgstr "Error fetching pause sets"

#: api_app/views/pause_set.py:96
msgid "Se obtuvo la informacion del conjunto de pausa de forma exitosa"
msgstr "Pause set information fetched successfully "

#: api_app/views/pause_set.py:113
msgid "Error al obtener el detalle del conjunto de pausa"
msgstr "Error fetching pause set detail"

#: api_app/views/pause_set.py:151
msgid "Error al crear el conjunto de pausa"
msgstr "Error creating pause set"

#: api_app/views/pause_set.py:179 api_app/views/pause_set.py:213
#: api_app/views/pause_set.py:255
msgid "No existe el conjunto de pausa"
msgstr "Pause set does not exist"

#: api_app/views/pause_set.py:184
msgid "Error al actualizar el conjunto de pausa"
msgstr "Error updating pause set"

#: api_app/views/pause_set.py:218
msgid "Error al eliminar el conjunto de pausa"
msgstr "Error deleting pause set"

#: api_app/views/pause_set.py:247
msgid "No existe la pausa"
msgstr "Pause does not exist"

#: api_app/views/pause_set.py:265
msgid "Error al crear la configuracion de pausa"
msgstr "Error creating pause configuration"

#: api_app/views/pause_set.py:293 api_app/views/pause_set.py:323
msgid "No existe la configuracion de pausa"
msgstr "Pause configuration does not exist"

#: api_app/views/pause_set.py:298
msgid "Error al eliminar la configuracion de pausa"
msgstr "Error deleting pause configuration"

#: api_app/views/pause_set.py:328
msgid "Error al actualizar la configuracion de pausa"
msgstr "Error updating pause configuration"

#: api_app/views/supervisor.py:108 api_app/views/supervisor.py:112
#: api_app/views/supervisor.py:115
msgid "Filtro \"type\" inválido"
msgstr "Invalid \"type\" filter"

#: api_app/views/supervisor.py:124
msgid "Filtro \"agent\" inválido"
msgstr "Invalid \"agent\" filter"

#: api_app/views/supervisor.py:128
msgid "Agente inexistente"
msgstr "Agent not found"

#: api_app/views/supervisor.py:139 api_app/views/supervisor.py:144
#: api_app/views/supervisor.py:147
msgid "Filtro \"status\" inválido"
msgstr "Invalid \"status\" filter"

#: api_app/views/supervisor.py:300 api_app/views/supervisor.py:346
msgid "ID Agenda incorrecto"
msgstr "Wrong Agenda Id"

#: api_app/views/supervisor.py:307
msgid "ID Agente incorrecto"
msgstr "Wrong Agent Id"

#: api_app/views/supervisor.py:315 api_app/views/supervisor.py:353
msgid "No tiene permiso para editar esta Agenda"
msgstr "You don't have permission to edit this Agenda"

#: api_app/views/supervisor.py:372
msgid "Recibidas"
msgstr "Received"

#: api_app/views/supervisor.py:373
msgid "Efectuadas"
msgstr "Dialed"

#: api_app/views/supervisor.py:374
msgid "Atendidas"
msgstr "Attended"

#: api_app/views/supervisor.py:375
msgid "Conectadas"
msgstr "Connected"

#: api_app/views/supervisor.py:376
msgid "No Conectadas"
msgstr "Not connected"

#: api_app/views/supervisor.py:377
msgid "Abandonadas"
msgstr "Abandon"

#: api_app/views/supervisor.py:378
msgid "Expiradas"
msgstr "Expired"

#: api_app/views/supervisor.py:379
msgid "Tiempo de Espera de Conexión(prom.)"
msgstr "Waiting time for connection(avg.)"

#: api_app/views/supervisor.py:380
msgid "Tiempo de Espera de Atención(prom.)"
msgstr "Waiting time for attention(avg.)"

#: api_app/views/supervisor.py:381
msgid "Tiempo de Abandono(prom.)"
msgstr "Abandon time(avg.)"

#: api_app/views/supervisor.py:384
msgid "Efectuadas Manuales"
msgstr "Manual dialed calls"

#: api_app/views/supervisor.py:385
msgid "Conectadas Manuales"
msgstr "Manual connected calls"

#: api_app/views/supervisor.py:386
msgid "No Conectadas Manuales"
msgstr "Manual not connected calls"

#: api_app/views/supervisor.py:387
msgid "Tiempo de Espera de Conexión Manuales(prom.)"
msgstr "Manual calls waiting time for connection(avg.)"

#: api_app/views/supervisor.py:513
msgid "Exportación de CSV en proceso"
msgstr "Exportation of CSV file in process"

#: api_app/views/supervisor.py:561
msgid "Exportación de contactados a .csv en proceso"
msgstr "Exportation of contacted calls to a .csv file in process"

#: api_app/views/supervisor.py:608
msgid "Exportación de calificados a .csv en proceso"
msgstr "Exportation of dispositioned calls to a .csv file in process"

#: api_app/views/supervisor.py:655
msgid "Exportación de no atendidos a .csv en proceso"
msgstr "Exportation of not attended calls to a .csv file in process"

#: api_app/views/supervisor.py:667
msgid "Finalizado"
msgstr "Finalized"

#: api_app/views/supervisor.py:669
msgid "Liberado"
msgstr "Liberated"

#: api_app/views/supervisor.py:671
msgid "Reservado"
msgstr "Reserved"

#: api_app/views/supervisor.py:755
msgid "Exportación de calificaciones a .csv en proceso"
msgstr "Exportation of dispositioned campaign calls to a .csv file in process"

#: api_app/views/supervisor.py:800
msgid "Exportación de Formularios de Gestiones a .csv en proceso"
msgstr "Exportation of Disposition Forms to a .csv file in process"

#: api_app/views/wombat_dialer.py:48 api_app/views/wombat_dialer.py:92
msgid "Refrescando. Espere al menos 15 segundos mientras finaliza el proceso."
msgstr "Refreshing. Wait at least 15 seconds while the process ends."
